### PR TITLE
[DS-416] Migrate to pnpm

### PR DESCRIPTION
**Pull request by victortrinh2**
Issue: https://workleap.atlassian.net/browse/DS-416

## Summary

To follow the stack of the Workleap organization. We are migrating from yarn to pnpm as a package manager to improve performance as a whole. Also migrating from lerna and npm-run-all to pnpm only.

## How to test

See if it still deploys on Netlify, Chromatic and passes the CI

- Is this testable with Jest or Chromatic screenshots? no

- Does this need an update to the documentation? yes

### CODE REVIEW

File: packages/components/src/disclosure/src/match.ts [hunk 1]
```diff
@@ -15,9 +15,5 @@ export function match<TValue extends string | number = string, TReturnValue = un
             .join(", ")}.`
     );
 
-    if (Error.captureStackTrace) {
```
Comment (alexasselin008) [inline packages/components/src/disclosure/src/match.ts:18]: That looks like a random change, can i have a one liner of what happened here?

File: PACKAGES_UPDATE.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ To udpate our packages use a package called ![npm-check-updates](https://www.npm
 In a terminal, use the followings commands
 
 - To list the available updates, type `npx --yes npm-check-updates`
```
Comment (alexasselin008) [inline PACKAGES_UPDATE.md:9]: it's a little bit extra, but could you add the 2 scripts 
```
        "list-outdated-deps": "pnpm outdated -r --format list",
        "update-outdated-deps": "pnpm update -r --latest \"!eslint\"",
```

We have them in hopper to update dependencies

File: .changeset/strong-pianos-attend.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+"@orbit-ui/transition-components": patch
+"@workleap/orbiter-ui": patch
+"@orbit-ui/transition-css": patch
+---
+
+Migrate from yarn and lerna to pnpm only
```
Comment (alexasselin008) [inline .changeset/strong-pianos-attend.md:7]: unecessary, we don't have to release for that. I would remove this file entirely

**Comment by alexasselin008** [comment]
The hopper preview don't seem to run on this PR? is it because it doesn't target master?

**Review by alexasselin008 — COMMENTED** [review]
The hopper preview don't seem to run on this PR? is it because it doesn't target master? 

Edit: It is. We need to remember that once this merge, we MIGHT have issues with netlify deployments

File: packages/components/src/disclosure/src/match.ts (hunk 1 above)
Comment (victortrinh2) [reply packages/components/src/disclosure/src/match.ts:18]: Ah I don't know why the typing didn't work anymore, doesn't exist on error

Comment (victortrinh2) [reply packages/components/src/disclosure/src/match.ts:18]: image

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**